src/components/package/show/package-show.js

```diff
@@ -169,6 +168,7 @@ class PackageShow extends Component {
           <Button
             buttonStyle="dropdownItem fullWidth"
             onClick={onEdit}
+            data-test-eholdings-package-edit-link
```

Comment:
Please add the test to check that the edit button click navigates to the correct edit page